### PR TITLE
Add Python 3.8 and Django 3.0 to the test matrix.

### PR DESCRIPTION
**Pull request by Natim**
Start fixing #151

### CODE REVIEW

**Comment by auvipy** [comment]
check the failing tests in travis

File: tox.ini [hunk 1]
```diff
@@ -1,18 +1,17 @@
...
     psycopg2-binary
-    111: Django>=1.11,<2.0
-    21: Django>=2.1,<2.2
-    22: Django>=2.2,<3.0
+    21: Django==2.1.*
+    22: Django==2.2.*
+    30: Django==3.0.*
+    trunk: https://github.com/django/django/archive/master.tar.gz
+ignore_outcome =
+    djangotrunk: True
```
Comment (jezdez) [inline tox.ini]: This should be `trunk`.

File: .travis.yml [hunk 2]
```diff
@@ -1,22 +1,23 @@
+version: ~> 1.0
```
Comment (jezdez) [inline .travis.yml]: Is `version` and `os` really needed?

**Comment by auvipy** [comment]
replace forece_text with force_str

File: example/testapp/models.py [hunk 3]
```diff
@@ -1,30 +1,21 @@
-# -*- coding: utf-8 -*-
 from django.db import models
-from django.utils.encoding import python_2_unicode_compatible
+from django.urls import reverse  # pylint: disable=ungrouped-imports
```
Comment (jezdez) [inline example/testapp/models.py]: Is the pylint skip comment still required?

**Comment by clintonb** [comment]
@Natim thanks for the contribution. Please squash your commits in the future to avoid extraneous commits on master. Releasing now.

**Comment by jezdez** [comment]
@clintonb I'll disable the non-squash merge option.

**Comment by jezdez** [comment]
Done, only rebase and squash merge allowed now.

**Comment by Natim** [comment]
> @Natim thanks for the contribution. Please squash your commits in the future to avoid extraneous commits on master. Releasing now.

What about using `git log --first-commit` instead of squashing and losing the story of the code?

**Comment by clintonb** [comment]
The pull request can serve as that story. Frankly, I have never encountered anyone who cares to see commits related to code review feedback (e.g., fixing a test that was broken by the PR, fixing typos).

I don’t want to dictate how people create commits. I prefer to do my squashing (using fixup) into relevant commits before pushing/merging. My goal with the previous request is to avoid the extraneous commits related to PR feedback. One commit would have told the entire story in this case.

**Pull request merged**